contracts/aave/AaveV2Contract.sol

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "hardhat/console.sol";
+import "../libraries/Errors.sol";
+import "../interfaces/IPoolV2.sol";
+import "../interfaces/IWETHGateway.sol";
+import "../libraries/PercentageMath.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+
+contract AaveWrapper is Ownable {
```

Comment:
Should we maybe also add description of what this contract does ? at the top ?

Reply:
Have noticed many defi protocols NOT following this and seems like a bad practise

Reply:
Note: Description should be very very detailed.